docs/features.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Feature flags
+---
+
+Below you can find a description of all the feature flags that are available in Busola dashboard and their configuration examples:
+
+> **TIP:** The list is ordered alphabetically.
+
+- **ADD_CLUSTER_DISABLED** – is used to enable or disable adding a cluster. If the function is enabled, you can only add your cluster from BTP cockpit using the KUBECONFIG_ID feature. Link to the cockpit is configured under cockpitUrl.
+
+Default settings:
+
+```bash
+"ADD_CLUSTER_DISABLED":{
+  "isEnabled": false,
+  "config": {
+    "cockpitUrl":"https://account.staging.hanavlab.ondemand.com/cockpit"
+  }
+},
+```
+
+- **ADDONS** – is used to show or hide the Addons view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"ADDONS ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"addons.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **API_GATEWAY** – is used to show or hide the Api Gateway view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
```

Comment:
API_GATEWAY is also used to determine if we should display Api Gateway list on functions and service details